packages/angular/projects/clr-angular/src/utils/popover/README.md

```diff
@@ -41,7 +41,7 @@ This is what it looks like from the implementing component perspective:
   <header class="header-4" role="heading">
     Header
     <button role="button" class="btn btn-link" clrPopoverCloseButton>
-      <clr-icon shape="close" size="36" class="is-inverse"></clr-icon>
+      <clr-icon shape="window-close" size="36" class="is-inverse"></clr-icon>
```

Comment:
SHould this be `<cds-icon shape="window-close" size="36" class="is-inverse"></cds-icon>`?